src/posix/platform/system.c

```diff
@@ -275,15 +275,17 @@ void otSysProcessDrivers(otInstance *aInstance)
         perror("select");
         exit(OT_EXIT_FAILURE);
     }
-
+    else if (rval >= 0)
```

Comment:
`>0`?

Reply:
`>= 0` required to handle timeout correct.